cmd/postgres_exporter/percona_compatibility_test.go

```diff
@@ -0,0 +1,142 @@
+//go:build manual
+// +build manual
```

Comment:
Probably we already can remove this old style tag `// +build manual`

Reply:
it get automatically added by Intellij, so I consider it still make sense. Are you referring to some recommendation? Can you please provide a link for that, so that I can share this practice with the team?

Reply:
https://go.dev/doc/go1.17 see section `//go:build lines`.
So if some tooling still adding this tag automatically for some reason - then we can leave it as is. But in general this format is deprecated.

Reply:
Ah, this project is on 1.17, and they deprecated this tag in 1.18. Well, ok then.